enigmatry-entry-blueprint-app/theming-setup.md

```diff
@@ -0,0 +1,233 @@
+```
+
+Our theme is now an empty object, and the default library theme is used.
+
+```scss
+@use 'entry-components/styles/generator';
+
+$custom-theme: (
+);
+
+@include generator.generate-from($custom-theme);
+
+```
+
+### Theme sections and properties
+
+Inside the `$custom-theme` map, you can add sections and define custom styles to suit specific preferences and needs.
+If a specific value is not defined in your project's custom theme, the value from the [default theme](#default-values-table) will be applied. <br>
+
+`$custom-theme` map includes 3 main sections:
+
+1. **general**
+- it contains styling for [**colors**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#2-colors), [**density**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#1-density), [**fonts**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#3-fonts), [**spacing**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#4-spacing), [**inputs**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#5-inputs), [**buttons**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#6-buttons)
+2. **tables**
+- it contains styling for [**cells**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#1-cells), [**rows**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#2-rows), [**contents**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#3-contents)
+3. **dialog**
+- it contains font size for the dialog [**title**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#1-title)
+
+> **_NOTE:_**  For detailed information and a deeper understanding of theming concepts and customization possibilities, visit our [**comprehensive guide here**](https://github.com/enigmatry/entry-angular-building-blocks/blob/master/libs/entry-components/configure-theming.md#theming-configuration-guide).
+
+Bellow is example of configuring custom theme:
+
+```scss
+@use 'entry-components/styles/generator';
+
+$custom-theme: (
+  // Configuration options
+  general: (
+    density: 1,
+    colors: (
+      primary-theme: #333
+    ),
+    fonts: (
+      body: (
+        size: 10px,
+      ),
+    )
+  ),
+  tables: (
+    cells: (
+      edge-gap: 4px
+    )
+  ),
+  dialogs: (
+    title: (
+      size: 20px
+    )
+  )
+);
+
+@include generator.generate-from($custom-theme);
+```
+
+### Fonts
+
+There are 2 possibilities of using fonts in theming:
+
+#### 1. Default Roboto font
+
```

Comment:
We already have this mentioned in more detailed guide. Just fill up some holes there (if any).